### DISCUSSION TITLE
How to modify the web URL of AdGuard Home? I have seted it to be the same as Openwrt by mistake, causing OP unable to log in

### DISCUSSION THREAD

**Discussion by higerfly** [AdguardTeam/AdGuardHome#6847] [Q&A]
将AdGuard Home的网址不小心也设置成与Openwrt相同192.168.1.2，现在这个网址只能登录AdGuard Home，而无法登录Openwrt，我想在AdGuard Home的网页中找到修改网址的方法，但是没有找到，请问还有其他方式修改AdGuard Home的网址吗？谢谢。
因为这个网址也不是在Openwrt中设置的，且目前无法登录Openwrt

**Comment by ghost** [comment]
You can modify the AGH access URL by following [these](https://adguard-dns.io/kb/adguard-home/faq/#webaddr) steps.
